### ISSUE TITLE
Setting minimum ages

### ISSUE THREAD

**Issue by jferrerobiol** [mmatschiner/snapp_prep#3]
Hi Michael,
I am running a divergence time analysis using your method described in Stange et al. 2016. I am setting the age constraint of the root with a normal distribution based on a previous study and I would like to add 3 additional minimum age constraints based on fossils like you did in the Stange et al. 2016 with the Neotropical Sea Catfish data. Reading the constraints options in the example.con.txt I can't find any way of only specifying a minimum constraint without assigning a particular distribution to it. What would be your recommended approach in that case?
Thank you very much,
Joan

**Comment by mmatschiner** [comment]
Hi Joan,

The way we did this in Stange et al. (and which I think is reasonable) is to use uniform prior distributions where the lower boundary is equal to the minimum age constraint and the upper boundary is set so high that it won't play a role at all. I think we used 1000 as upper boundary. The normal distribution on the root should completely prevent the chain from reaching the upper boundary of the uniform priors, meaning that then only the  lower boundaries (= the minimum ages) of the uniform priors will have an effect.

Best,
Michael
